app/assets/stylesheets/banner.scss

```diff
@@ -0,0 +1,11 @@
+.banner {
+  margin-bottom: 2rem;
```

Comment:
This one is tricky because the margin is needed in some sections but not in other sections :thinking:. I guess it's better to have too much margin in places like the proposals section rather than too little in places like the help page, so I'm fine with this change :+1:.